src/Keboola/StorageApi/Client.php

```diff
@@ -1468,6 +1467,7 @@ public function uploadSlicedFile(array $slices, FileUploadOptions $options, File
                     $slices[$key] = $gzFilePath;
                 }
             }
+            $newOptions->setFileName($newOptions->getFileName() . '.gz');
```

Comment:
Asi by to chtělo doplnit i do testů např. [tady](https://github.com/keboola/storage-api-php-client/blob/6b01f17d008a1a6161306ba24609252024e928b8/tests/Common/SlicedFilesUploadTest.php#L20)  přidat varinatu  s `compress=true`. 

A tady přidat kontrolu jména https://github.com/keboola/storage-api-php-client/blob/6b01f17d008a1a6161306ba24609252024e928b8/tests/Backend/CommonPart2/UploadSlicedFileTest.php#L29

Reply:
Testy, no jasně, to jsem úplnej idiot, že jsem je vynechal. Měl jsem v tu chvíli takovej hackerskej mindset a testoval a benchmarkoval jsem to přes docker runner na testingu a lokálně v output mappingu. 

Jsem se zase do něčeho pustil, že to rychle odbavím, a pojedu dál v původních úkolech 👿Jdu na to sednout pořádně.

Reply:
j jinak bych to nechal tak jak to máš teď, jenom ten test upravit/přidat.